nvdaHelper/archBuild_sconscript

```diff
@@ -95,7 +95,8 @@ env.Append(
 env.Append(CCFLAGS=[
 	'/W3',
 	'/WX',
-	'/std:c++17'
+	'/std:c++17',
+	'/await'

```

Comment:
This enables co-routine support for all of the modules, I think a comment is warranted here to explain what needs this specifically.

Reply:
I think given this needs new compiler flags, and the rest of our code doesn't (yet) I'd like this PR to look at separating this code  to a separate module / dll.

Reply:
The code is already in a separate dll.
Note that before this pr, we had a separate win 10 environment that added compiler flags to support C++/CX, but I think changing parameters at the sconscript level is ok.
Having said that, I made /await global as I think there's potential in using co-routines more widely in our codebase. For example, @michaelDCurran used APCs in #11431 and #11425, but co-routines could have been appropriate here as well.

Reply:
Yes, sorry I missed that.

`/await` may be useful in other areas too. But I'd like that to decided on a case by case basis. Given this hasn't actually been finalized in the C++ standard yet (but included in draft for C++20), we should be cautious about how we use it. I think using it via WinRT should be fine, they will have handle any adjustments as the compiler adjusts. This recent blog post talks about the differences between `/await` and C++20: https://devblogs.microsoft.com/cppblog/c-coroutines-in-visual-studio-2019-version-16-8/

 It would be useful to call this out (and the blog) on the PR description.

Reply:
After reading that blog post more carefully, I think it is worth considering using `/std:c++latest` rather than `/await`.